frontend/elements/src/entry/home/sections/logout-section.ts

```diff
@@ -80,13 +78,13 @@ import { MEDIA_UI } from '@utils/path';
 
          <div class="lines">
 
-        <column-list class="line" text_line="${STR_LINE1}" size="medium"></column-list>
-     <column-list class="line" text_line="${STR_LINE2}" size="medium"></column-list>
-     <column-list class="line" text_line="${STR_LINE3}" size="medium"></column-list>
+        <column-list class="line" text_line="${STR_IFYOU}" size="medium"></column-list>
+     <column-list class="line" text_line="${STR_LOG}" size="medium"></column-list>
+     <column-list class="line" text_line="${STR_DOYOU}" size="medium"></column-list>
 
         </div>
          <div class="inside-wrapper">
-         <button-rect size="large" color="blue" class="button">${STR_LABEL}</button-rect>
+         <button-rect size="large" color="blue" class="button">${STR_LOGOUT}</button-rect>
```

Comment:
these need to be slotted in from the the outside because we have to attach functionality